plugins/balloontoolbar/skins/moono-lisa/balloontoolbar.css

```diff
@@ -22,50 +15,6 @@ For licensing, see LICENSE.md or http://ckeditor.com/license
 	border-color: #d1d1d1 transparent;
```

Comment:
This is also in [`default.css`](https://github.com/ckeditor/ckeditor-dev/blob/7ecc15bc26aef53fadb7f3ec342510ca2d736236/plugins/balloontoolbar/skins/default.css#L60).